src/main/java/ca/corefacility/bioinformatics/irida/config/services/IridaPluginConfig.java

```diff
@@ -33,7 +34,7 @@
 	 */
 	@Bean(name = "iridaPipelinePlugins")
 	public IridaPluginList iridaPipelinePlugins() {
-		DefaultPluginManager pluginManager = new DefaultPluginManager(PIPELINE_PLUGIN_PATH);
+		DefaultPluginManager pluginManager = new DefaultPluginManager(Paths.get(pipelinePluginPath));
```

Comment:
Since this path is now configurable, can you add a debug level logging statement just above this with "Scanning <path> for pipeline plugins." for verification in the logs?

Reply:
I included the path in the message